etc/ci/performance/test_perf.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TP5N_SOURCE="http://people.mozilla.org/~jmaher/taloszips/zips/tp5n.zip"
+TP5N_PATH="page_load_test/tp5n.zip"
+if [[ ! -f "${TP5N_PATH}" ]]; then
+    wget "${TP5N_SOURCE}" -O "${TP5N_PATH}"
+fi
+unzip -o "${TP5N_PATH}" -d "$(dirname "${TP5N_PATH}")"
+
+virtualenv venv --python="$(which python3)"
+PS1="" source venv/bin/activate
+# `PS1` must be defined before activating virtualenv
+pip install "treeherder-client>=3.0.0"
+
+mkdir -p servo
+mkdir -p output
+./git_log_to_json.sh > servo/revision.json && \
+cp -r ../../../resources servo && \
```

Comment:
If we are running `../../../target/release/servo` in `runner.py`, we probably don't need to copy this folder.